vignettes/adsl.Rmd

```diff
@@ -241,19 +241,37 @@ is to be imputed to the first of the month, then set `date_imputation = "FIRST"`
 
 ### Disposition Status (e.g. `EOSSTT`) {#disposition_status}
 
-The function `derive_var_disposition_status()` can be used to derive a disposition status at a specific
+The function `derive_var__merged_cat()` can be used to derive a disposition status at a specific
 timepoint. The relevant disposition variable (`DS.DSDECOD`) is selected by adjusting the filter 
 parameter and used to derive `EOSSTT`.
 
 To derive the End of Study status (`EOSSTT`), a call could be:
 
+```{r eval=TRUE, echo=FALSE}
+# Setting up the values of the source variable as input for the cat_fun argument in derive_var_merged_cat function
+format_dsdecod <- function(dsdecod_cat) {
+  case_when(
+    dsdecod_cat %in% c("SCREEN FAILURE", "SCREENING NOT COMPLETED") ~ "NOT STARTED",
+    dsdecod_cat %in% "COMPLETED" ~ "COMPLETED",
+    dsdecod_cat != "COMPLETED" | is.na(dsdecod_cat) ~ "DISCONTINUED",
+    TRUE ~ "ONGOING"
+  )
+}
+```
+
+
```

Comment:
I think this should be exposed to the reader.  Currently it is hidden with `echo=FALSE`

Reply:
![image](https://user-images.githubusercontent.com/10111024/216786089-fc87426e-d3c9-455c-8360-4cb2fd538d0a.png)

Reply:
We also want to point to it more like you do in the next section